### PR TITLE
Updates for WP v3.8.0

### PR DESCRIPTION
**Pull request by joshcanhelp**
- Add new settings and fix mis-wording on the Configuration page
- Change function/hook names and add new hooks to Extending page
- Add invalid state troubleshooting page

### CODE REVIEW

File: articles/cms/wordpress/configuration.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ First, we'll check for the Application created for your WordPress site.
 
 In order for your WordPress site to perform certain actions on behalf of your Auth0 tenant, you'll need to authorize the Application created above to access the Management API.
 
+1. Make sure your Application allows the Client Credential grant (step 11 above)
```
Comment (joshcanhelp) [inline articles/cms/wordpress/configuration.md:86]: I know this is duplicative but I often find myself linking directly to this section during support and this is a key step, often the one that's missed.

File: articles/cms/wordpress/configuration.md [hunk 2]
```diff
@@ -188,14 +190,19 @@ All sites in a WordPress multi-site network will use the same constant value mak
...
 
 * **Single Sign On (SSO):** Enables SSO on your WordPress, allowing users to log in once and be automatically logged into any of your sites which use Auth0. For more information, see [What is SSO?](/sso). Activating this setting will attempt to turn on "Use Auth0 instead of the IdP to do Single Sign On" in the Application settings in the Auth0 dashboard (requires a valid API token to make this change). Option name is `sso`.
 
-* **Single Logout:** Enable this option for Single Logout. For more information, see [What is Single Log Out?](/sso/single-sign-on#what-is-single-log-out-). This will be hidden (and automatically disabled) if SSO is turned on. Option name is `singlelogout`.
+* **Single Logout:** Enable this option for Single Logout. For more information, see [What is Single Log Out?](/sso/single-sign-on#what-is-single-log-out-). This will be hidden (and automatically disabled) if SSO is turned off. Option name is `singlelogout`.
```
Comment (joshcanhelp) [inline articles/cms/wordpress/configuration.md:197]: 🙈

File: articles/cms/wordpress/configuration.md [hunk 3]
```diff
@@ -260,10 +269,6 @@ All sites in a WordPress multi-site network will use the same constant value mak
 
 * **Migration IPs Whitelist:** Only requests from listed IPs will be allowed access to the migration webservice. Option name is `migration_ips_filter`.
 
-* **Auto Login:** Redirects the `wp-login.php` page to the Universal Login Page for authentication using all active Connections for this Application. Option name is `auto_login`.
```
Comment (joshcanhelp) [inline articles/cms/wordpress/configuration.md:263]: Moved and renamed

File: articles/cms/wordpress/extending.md [hunk 4]
```diff
@@ -410,15 +410,72 @@ Use this filter to change the prefix for the constant used to override plugin se
...
 
 	// Prefix the prefix.
 	return 'PREFIX_' . $prefix;
 }
-// add_filter( 'auth0_settings_constant_prefix', 'auth0_theme_hook_settings_constant_prefix' );
-``` 
+add_filter( 'auth0_settings_constant_prefix', 'auth0_docs_hook_settings_constant_prefix' );
+```
+
+### auth0_authorize_url_params
```
Comment (joshcanhelp) [inline articles/cms/wordpress/extending.md:423]: New filters here and below

File: articles/cms/wordpress/invalid-state.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+---
```
Comment (joshcanhelp) [inline articles/cms/wordpress/invalid-state.md:1]: New page, only linked from the Troubleshooting doc

File: articles/cms/wordpress/invalid-state.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+description: Troubleshooting invalid state errors in the Login by Auth0 WordPress plugin.
+topics:
+    - wordpress
+    - cms
+    - state
+contentType: reference
```
Comment (joshcanhelp) [inline articles/cms/wordpress/invalid-state.md:7]: Is this correct? Just copy-pasted from another page 🤷‍♂️

File: articles/cms/wordpress/invalid-state.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+useCase:
```
Comment (joshcanhelp) [inline articles/cms/wordpress/invalid-state.md:8]: Same question ...

**Comment by erjohnson** [comment]
[Invalid State Errors preview](https://auth0-docs-content-pr-6919.herokuapp.com/docs/cms/wordpress/invalid-state)

**Review by erjohnson — APPROVED** [review]

**Comment by joshcanhelp** [comment]
Thanks @erjohnson!

**Pull request merged**